tests/functions/test_buffer.py

```diff
@@ -59,7 +59,7 @@ def test_buffer_standalone_noise_function_in_array(self, benchmark):
         expected_val = [[24, 4.693117564500052, 46], [17, 7.744647273059847, 29], [10, 11, 12]]
         for v_v, v_e in zip(val, expected_val):
             for v, e in zip(v_v, v_e):
```

Comment:
Can the zip iterations be replaced by a single np.testing.assert_allclose?

Reply:
Fixing